### DISCUSSION TITLE
"Linking" 2 Cask installs together?

### DISCUSSION THREAD

**Discussion by cycomachead** [Homebrew/discussions#4010] [Casks]
I am wondering if there's an easy way to link the installation or update of two casks. Specifically, I am interested in always updating `google-chrome` and `chromedriver` together since the versions need to be paired.

**Comment by p-linnane** [comment]
There is no way to do this specifically, but you can always run `brew upgrade --greedy` which will update all casks including those that autoupdate like `google-chrome`.

**Comment by cycomachead** [comment]
Thanks, that makes sense.
